### ISSUE TITLE
Customer Onboarding for Turborepo Launch

### ISSUE THREAD

**Issue by becca** [vercel/turborepo#110]
After our conversation with the EPD leadership team this morning we are committing to finalizing a plan for how to better onboard customers to Turborepo and what that looks like when they need a Vercel account. The following is the agreed upon scope for this work which was discussed with @becca @jaredpalmer and @gaspar09. 

We can accomplish the scope listed below in time for Dec 9th but have requests listed below for what we need in order to get there, specifically having some Design help and open questions answered.

## Scope for Launch
We will be focused on making sure the initial customer signup flow is more seamless as right now the connection and workflow between Turborepo is lacking. 

**The following is scope we will be prioritizing for the launch:**

#### Website Updates:
**TLDR**: The following is the flow that a user will encounter when they attempt to run Turbo from their CLI without a vercel account. Instead of an error message to signup for a Vercel account we'll update the language to be friendlier and nudge them to a Turborepo+Vercel workflow to signup for an account, connect Turborepo and return to the CLI after.

1. dedicated landing page for Turborepo new signups which will live at vercel.com/signup/?ref=turbo
   - params added to url to denote this is a turbo LP 
        - _what params does marketing and product need here for metrics?_ we want to know how many users are coming to turbo to signup. 
2. user has the same options for logging in that we have today on `/signup` (github, gitlab, bitbucket, email)
3. after a user has successfully logged in (before they hit https://vercel.com/new?onboarding=true, which displays the import options and templates) they will **instead** see a screen with clear directions of how to use the CLI and connect Turborepo to their newly created Vercel account. 🤝 
    - we will have to add another parameter into the URL here to denote that this successful signup is from Turborepo.

#### CLI Updates:
1. We will update the language on remote caching to be more specific as to what `link` does. (remove ref to "contain" and expand a bit around what linking does)
2. `turbo link` -  add some friendly friction for customers to confirm that they are okay with linking Turborepo to all their repositories within their account.

## Open Questions
-  Does Turbo no longer need to share the same token as Vercel CLI? 
    - @southpolesteve is this something we need to update before the 9th? If this is a different token then how are we getting that?

## Our Ask
In order to meet the the current launch date of December 9th we will need folks to please answer the `open questions` above and we need a dedicated Design resource for the landing pages to be customer-ready.

## Scope for Later (After Launch)
- When a user creates a team to use Turborepo (for said team) the flow on Vercel for what the customer does after creating said team on Vercel is not clear. 
   - We will design a clear workflow for a Turborepo team creation and onboarding flow.
